src/app/appmarket/appinstance/appinstance/appinstance.component.html

```diff
@@ -206,16 +206,19 @@ <h4 style="text-align: center">{{'APP_INSTANCES.MANUAL_UPDATE.HEADER' | translat
         <div class="nmaas-modal-body">
             <div class="" style="height: 150px">
                 <p>{{'APP_INSTANCES.MANUAL_UPDATE.BODY' | translate}}</p>
-                <div>
+                <div *ngIf="appVersions.length > 0">
                     <p-dropdown id="manualUpdate" [options]="appVersions" optionLabel="version" optionValue="version"
                                 placeholder="Select version" emptyMessage="No version available"
                                 [disabled]="appVersions.length === 0" [(ngModel)]="selectedVersion" name="manualUpdate"
                                 ngDefaultControl></p-dropdown>
                 </div>
+                <div *ngIf="appVersions.length === 0" class="font-bold">
+                    {{'No other versions available'}}
```

Comment:
Replace this static text with a label

Reply:
https://github.com/nmaas-platform/nmaas-platform/pull/172